src/base/constants.py

```diff
@@ -44,3 +44,16 @@ def _IsDevAppServer():
 DEBUG = IS_DEV_APPSERVER
 
 TEMPLATE_DIR = os.path.sep.join([os.path.dirname(__file__), '..', 'templates'])
+
+AUTHORIZATION_METHOD = 'basic_auth'
```

Comment:
This is not the right place to do runtime application configuration. The file header explicitly states that this is constants, not configuration variables.